### PR TITLE
[Draft] Build aarch64

### PR DESCRIPTION
**Pull request by mwittgen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -50,11 +50,11 @@ outputs:
         # above, we need most of these at build time, so they are mostly
         # host-like
         - boost =1.74
-        - cfitsio =3
+        - cfitsio =4
         - eigen =3.3
         - eups
         - firefly-client
-        - galsim =2.2
+        - galsim =2.3
```
Comment (beckermr) [inline recipe/meta.yaml:57]: @ktlim will need to approve these pin changes.

Comment (mwittgen) [reply recipe/meta.yaml:57]: This is meant as a draft with a tentative version number `0.8.0` for testing. 
`galsim` and `cfitsio` can be unpinned and would currently resolve to the versions above. Pinning to the previous versions would result in a conflict for `aarch64`

Comment (beckermr) [reply recipe/meta.yaml:57]: cfitsio is not at 4 yet. we have an ongoing migration: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L393

**Comment by mwittgen** [comment]
Closing and basing a new draft on version 1.0.0

**Comment by timj** [comment]
Is there a reason you didn't rebase and force push to reuse this pull request? I don't think we need to create a new PR every time a rebase is needed do we?

**Comment by mwittgen** [comment]
I am only human and out of the 111 decisions I make in a day there might be 3 bad ones.

**Pull request closed without merging**